src/main/java/li/cil/tis3d/common/init/Items.java

```diff
@@ -55,6 +58,13 @@ public static void register(final ProxyCommon proxy) {
             modules.put(module, proxy.registerModule(module));
         }
 
+        // Prevent stacking of ROM modules to avoid inconsistencies between new
+        // and already programmed modules, and allow sneak clicking with it to
+        // program it to the clicked RAM/ROM module.
+        ((ItemModule) modules.get(Constants.NAME_ITEM_MODULE_READ_ONLY_MEMORY)).
```

Comment:
Looks like the `setShareTag` in `ItemModule` is missing, mind backporting that, too?